app/assets/stylesheets/pageflow/editor/files_explorer.css.scss

```diff
@@ -33,6 +33,12 @@
 
   .entries_panel {
     width: 20%;
+
+    li.blank_slate {
+      text-align: center;
+      padding: 10px 5px 20px 5px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 5px 20px` instead of `10px 5px 20px 5px`